cpp/src/robotick/framework/data/TelemetryServer.cpp

```
@@ -75,521 +79,237 @@ namespace robotick
-				}, 100);
-
-				loadInitialData();
-			</script>
-			</body>
-			</html>
-			)";
-
-		res.status_code = 200;
-		res.content_type = "text/html";
-		res.body.set_from_string(telemetry_html);
+		char buffer[64];
```

Clang (14.0.6)

[warning] do not declare C-style arrays, use std::array<> instead

(cppcoreguidelines-avoid-c-arrays,modernize-avoid-c-arrays)